### DISCUSSION TITLE
Error in booting

### DISCUSSION THREAD

**Discussion by LunaBaker** [raspiblitz/raspiblitz#4345] [Q&A]
Hi Guys!

I'm having some trouble booting Raspiblitz onto my brand new RPi.
I followed the instructions from the github, checked with several youtube videos. And I haven't seen other people having the same issue.
I wrote the SD image onto the SD card I used initially Ethecher, after I used rufus and finaly I used the official RPi writer to boot with Raspiblitz.
I made sure the ethernet cable is functional, the screen & the SSD cable they all work fine.
I tried installing the official recommended RPi OS just to test that nothing was wrong with my RPi and it booted normally.

All I get is a grey screen, and then a moving line. When I plug a normal size monitor there is a white screen with gradual colors but then it disappears and nothing.

Unfortunately I can get a debug report because I can't even start RaspiBlitz.
For the SD Image, used 7zip to extract the downloaded file and the SD image file ends up going up to 17 GB.
The file was : raspiblitz-fat-v1.10.0-2023-09-22.img

(See picture)

Any help would be appreciated!

Thanks!
 
![2](https://github.com/raspiblitz/raspiblitz/assets/155334597/b79219db-e37e-482f-a545-67ef7bf175fd)
![1](https://github.com/raspiblitz/raspiblitz/assets/155334597/a382ff41-1032-4e6c-9504-756eb4c13b79)

**Comment by rootzoll** [comment]
The LCD seems to be a model i dont have seen before so I am not sure if it will work with the RaspiBlitz LCD drivers. To use HDMI you need to switch from lcd to hdmi-mode. https://github.com/raspiblitz/raspiblitz/blob/v1.10/FAQ.md#can-i-run-the-raspiblitz-without-a-displaylcd
